src/main/java/org/example/web/MainServlet.java

```diff
@@ -67,4 +70,23 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
             }
         }
     }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) {
+        try {
+            developerService.addDeveloper(req);
```

Comment:
Why do not use the `pageMapService.getResponse`?

Reply:
fixed